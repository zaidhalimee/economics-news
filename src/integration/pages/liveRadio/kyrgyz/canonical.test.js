/**
 * @service kyrgyz
 * @pathname /kyrgyz/bbc_kyrgyz_radio/liveradio
 */

import runCanonicalTests from '../canonicalTests';

describe('Canonical Kyrgyz Live Radio Page', () => {
  runCanonicalTests(service);
});
