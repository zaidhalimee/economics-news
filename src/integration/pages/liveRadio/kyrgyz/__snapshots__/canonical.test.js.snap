// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical Kyrgyz Live Radio Page A11y I can see the skip to content link 1`] = `"Сайтка өтүү"`;

exports[`Canonical Kyrgyz Live Radio Page Analytics ATI tracking pixel excluding src 1`] = `
<img
  alt=""
  height="1px"
  style="position:absolute"
  width="1px"
/>
`;

exports[`Canonical Kyrgyz Live Radio Page Analytics ATI tracking pixel hostname 1`] = `"https://logws1363.ati-host.net/"`;

exports[`Canonical Kyrgyz Live Radio Page Analytics ATI tracking pixel search params 1`] = `
[
  {
    "s": "598343",
  },
  {
    "s2": "58",
  },
  {
    "p": "kyrgyz.bbc_kyrgyz_radio.liveradio.page",
  },
  {
    "x1": "[urn:bbc:ares:ws_media:live:bbc_kyrgyz_radio]",
  },
  {
    "x2": "[responsive]",
  },
  {
    "x3": "[news-kyrgyz]",
  },
  {
    "x4": "[ky]",
  },
  {
    "x7": "[player-live]",
  },
  {
    "x8": "[simorgh-nojs]",
  },
  {
    "x9": "[Би-Би-Си%20Кыргыз%20кызматынын%20радиосу%20-%20BBC%20News%20Кыргыз%20КызMATы]",
  },
]
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 1`] = `
{
  "text": "BBC News, Кыргыз КызMATы",
  "url": "/kyrgyz",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 2`] = `
{
  "text": "Эмнеге BBC News ишенсе болот?",
  "url": "https://www.bbc.com/kyrgyz/institutional-49677275",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 3`] = `
{
  "text": "Колдонуу эрежелери",
  "url": "https://www.bbc.com/kyrgyz/institutional-38157280",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 4`] = `
{
  "text": "Би-Би-Си жөнүндө",
  "url": "https://www.bbc.com/kyrgyz/institutional-38157281",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 5`] = `
{
  "text": "Купуялык",
  "url": "https://www.bbc.com/kyrgyz/institutional-38157282",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 6`] = `
{
  "text": "Cookies",
  "url": "https://www.bbc.com/usingthebbc/cookies/",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 7`] = `
{
  "text": "Би-Би-Си менен байланышыңыз",
  "url": "https://www.bbc.co.uk/kyrgyz/send/u50853445",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 8`] = `
{
  "text": "Other Languages",
  "url": "https://www.bbc.com/ws/languages",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer Anchors should match text and url 9`] = `
{
  "text": "Башка интернет сайттардын мазмуну боюнча биздин позиция.",
  "url": "https://www.bbc.com/editorialguidelines/guidance/feeds-and-links",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/kyrgyz"`;

exports[`Canonical Kyrgyz Live Radio Page Footer Paragraphs should match text 1`] = `"© 2024 BBC. Би-Би-Си сырткы интернет сайттардын мазмуну үчүн жооптуу эмес. Башка интернет сайттардын мазмуну боюнча биздин позиция."`;

exports[`Canonical Kyrgyz Live Radio Page Header I can see the branding 1`] = `"BBC News, Кыргыз КызMATы"`;

exports[`Canonical Kyrgyz Live Radio Page Header Navigation link should match text and url 1`] = `
{
  "text": "Башталгыч бет",
  "url": "/kyrgyz",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Header Navigation link should match text and url 2`] = `
{
  "text": "Кыргызстан",
  "url": "/kyrgyz/topics/cz74kjpyk07t",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Header Navigation link should match text and url 3`] = `
{
  "text": "Дүйнө",
  "url": "/kyrgyz/topics/ck2l9z012nkt",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Header Navigation link should match text and url 4`] = `
{
  "text": "САПАР",
  "url": "/kyrgyz/topics/c6z8lg83w7mt",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Header Navigation link should match text and url 5`] = `
{
  "text": "Журнал",
  "url": "/kyrgyz/topics/czp8pjrkgp0t",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Header Navigation link should match text and url 6`] = `
{
  "text": "Подкасттар",
  "url": "/kyrgyz/podcasts/p0c80v81",
}
`;

exports[`Canonical Kyrgyz Live Radio Page Main heading should match text 1`] = `"Би-Би-Си Кыргыз кызматынын радиосу"`;

exports[`Canonical Kyrgyz Live Radio Page Media Loader renders a valid container 1`] = `
<figure
  class="bbc-ucozvf"
  data-e2e="media-loader__container"
>
  <div
    class="bbc-1eq7znh"
    data-e2e="media-player"
  />
</figure>
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 1`] = `
{
  "sizes": null,
  "url": "http://localhost:7080/kyrgyz/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 2`] = `
{
  "sizes": "72x72",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-72x72.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 3`] = `
{
  "sizes": "96x96",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-96x96.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 4`] = `
{
  "sizes": "128x128",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-128x128.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 5`] = `
{
  "sizes": "144x144",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-144x144.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 6`] = `
{
  "sizes": "152x152",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-152x152.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 7`] = `
{
  "sizes": "180x180",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-180x180.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 8`] = `
{
  "sizes": "192x192",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-192x192.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 9`] = `
{
  "sizes": "384x384",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-384x384.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Apple Touch Icon should match attributes 10`] = `
{
  "sizes": "512x512",
  "url": "https://static.files.bbci.co.uk/ws/simorgh-assets/public/kyrgyz/images/icons/icon-512x512.png",
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO Canonical link 1`] = `"http://localhost:7080/kyrgyz/bbc_kyrgyz_radio/liveradio"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical Kyrgyz Live Radio Page SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Heading level 1 1`] = `"Би-Би-Си Кыргыз кызматынын радиосу"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Lang attribute 1`] = `"ky"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Linked data should match text 1`] = `
{
  "@context": "http://schema.org",
  "@graph": [
    {
      "@type": "RadioChannel",
      "image": {
        "@type": "ImageObject",
        "height": 576,
        "url": "https://news.files.bbci.co.uk/ws/img/logos/og/kyrgyz.png",
        "width": 1024,
      },
      "inLanguage": {
        "@type": "Language",
        "alternateName": "ky",
        "name": "Kyrgyz",
      },
      "mainEntityOfPage": {
        "@id": "http://localhost:7080/kyrgyz/bbc_kyrgyz_radio/liveradio",
        "@type": "WebPage",
        "name": "Би-Би-Си Кыргыз кызматынын радиосу",
      },
      "url": "http://localhost:7080/kyrgyz/bbc_kyrgyz_radio/liveradio",
    },
  ],
}
`;

exports[`Canonical Kyrgyz Live Radio Page SEO OG description 1`] = `"Эл аралык жана жергиликтүү жаңылыктар, аналитика, кызыктуу материалдар кыргыз тилинде"`;

exports[`Canonical Kyrgyz Live Radio Page SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/kyrgyz.png"`;

exports[`Canonical Kyrgyz Live Radio Page SEO OG image alt 1`] = `"BBC News Кыргыз КызMATы"`;

exports[`Canonical Kyrgyz Live Radio Page SEO OG locale 1`] = `"ky-KG"`;

exports[`Canonical Kyrgyz Live Radio Page SEO OG site name 1`] = `"BBC News Кыргыз Кызматы"`;

exports[`Canonical Kyrgyz Live Radio Page SEO OG title 1`] = `"Би-Би-Си Кыргыз кызматынын радиосу - BBC News Кыргыз Кызматы"`;

exports[`Canonical Kyrgyz Live Radio Page SEO OG type 1`] = `"website"`;

exports[`Canonical Kyrgyz Live Radio Page SEO OG url 1`] = `"http://localhost:7080/kyrgyz/bbc_kyrgyz_radio/liveradio"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Page title 1`] = `"Би-Би-Си Кыргыз кызматынын радиосу - BBC News Кыргыз Кызматы"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Robots meta tag 1`] = `"noodp, noydir, max-image-preview:large"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Twitter creator 1`] = `"@bbckyrgyz"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Twitter description 1`] = `"Эл аралык жана жергиликтүү жаңылыктар, аналитика, кызыктуу материалдар кыргыз тилинде"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Twitter image alt 1`] = `"BBC News Кыргыз КызMATы"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/kyrgyz.png"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Twitter site 1`] = `"@bbckyrgyz"`;

exports[`Canonical Kyrgyz Live Radio Page SEO Twitter title 1`] = `"Би-Би-Си Кыргыз кызматынын радиосу - BBC News Кыргыз Кызматы"`;

exports[`Canonical Kyrgyz Live Radio Page Summary should match text 1`] = `"Эл аралык жана жергиликтүү жаңылыктар, аналитика, кызыктуу материалдар кыргыз тилинде"`;
